### PR TITLE
Blog Post: Seventh Wave of Nostr Grants

### PR DESCRIPTION
**Pull request by Arvin21M**
**Build Preview:**
- [/blog/7th-wave-of-nostr-grants](https://os-website-git-7th-wave-nostr-grants-open-sats.vercel.app/blog/7th-wave-of-nostr-grants)

**Social Preview:**
- image

### CODE REVIEW

**Review by btcschellingpt — APPROVED** [review]
Reads well and all links checked and working; good work on the refocus of JeffG's activities within this wave.

Good to go!

**Comment by erskingardner** [comment]
I would remove the last paragraph of the section on me. The one on ostrich.work and ndk. I'm still helping on ndk but my focus is almost entirely on the messaging stuff

**Comment by btcschellingpt** [comment]
> I would remove the last paragraph of the section on me. The one on ostrich.work and ndk. I'm still helping on ndk but my focus is almost entirely on the messaging stuff

That was my takeaway from listening to your sessions in Riga Jeff! :-)

**Comment by dergigi** [comment]
Made some edits, so I'd appreciate a quick sanity check from one of the @OpenSats/volunteers :pray:

File: data/blog/7th-wave-of-nostr-grants.mdx [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+### futr
+
+Futr is a decentralized nostr client written in Haskell, focusing on public and
+private groups ([NIP-29]) and private direct messages ([NIP-17]). The project
+positions itself more like Telegram than Twitter. It's a native app that doesn't
+rely on a browser engine and is designed to cross-compile to Linux, Windows,
+macOS, Android, and iOS. Plans include refactoring for better performance,
+improving the user interface (possibly switching to Qt), implementing group and
+private messaging, and integrating features like voice/video calls and Lightning
+Network zaps to enhance secure and decentralized communication.
```
Comment (Bayernatoor) [inline data/blog/7th-wave-of-nostr-grants.mdx:91]:
```suggestion
Network zaps to enhance secure and decentralized communication.
```

Comment (Bayernatoor) [inline data/blog/7th-wave-of-nostr-grants.mdx:91]:
```suggestion
Network zaps with the aim of improving security and further decentralizing communications.
```

**Review by Bayernatoor — CHANGES_REQUESTED** [review]
Added 1 suggestion, the ending of that sentence seemed a bit off. 

Besides that all links work and everything else reads well! Good job guys. Awesome projects being supported here :orange_heart:

**Review by dergigi — APPROVED** [review]
Looks good to me. Let's ship it 🐿️

**Pull request merged**